artifacts/index.ts

```diff
@@ -69,7 +69,6 @@ export {
   ExchangeIssuanceLibraryContract,
   ExchangeIssuanceModuleContract,
   FailAuctionLibraryContract,
-  FactoryUtilsLibraryContract,
```

Comment:
Any reason to delete?

Reply:
It doesnt exist anymore? Failed previously when compiling